.github/scripts/ci_tests.sh

```diff
@@ -59,6 +60,7 @@ do
             echo -e "::group::Building sample for ESPRESSIF ESP32 port"
             sample_build "ESPRESSIF" "esp32" "build"
```

Comment:
how about calling :
sample_build "ESPRESSIF" "esp32" "build_esp32"

sample_build "ESPRESSIF" "aziotkit" "build_aziotkit"


same thing we do for ST different boards?